### PR TITLE
Fix broken link to Logstash Reference 

### PR DESCRIPTION
**Pull request by karenzone**

### CODE REVIEW

**Comment by karenzone** [comment]
@logstashmachine bump patch

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 11.9.2
+  - Fix broken link to Logstash Reference  [#1085](https://api.github.com/repos/logstash-plugins/logstash-output-elasticsearch/pulls/1085)
```
Comment (karenzone) [inline CHANGELOG.md:2]:
```suggestion
  - Fix broken link to Logstash Reference  [#1085](https://api.github.com/repos/logstash-plugins/logstash-output-elasticsearch/pulls/1085)
```

Comment (karenzone) [inline CHANGELOG.md:2]:
```suggestion
  - Fix broken link to Logstash Reference  [#1085](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1085)
```

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚗

**Comment by karenzone** [comment]
Published at 11.9.2: https://rubygems.org/gems/logstash-output-elasticsearch/versions/11.9.2-java

**Pull request merged**